### PR TITLE
Bugfix/01 robot reconfigure

### PR DESCRIPTION
**Pull request by skyler-ferrante**

### CODE REVIEW

File: src/main/java/frc/subsystem/Shooter.java [hunk 1]
```diff
@@ -37,6 +39,19 @@ void execute() {
...
+		FlywheelMotorBeta.set(ControlMode.PercentOutput, 0.64);
+		System.out.println(shooterSensors.getShooterAngleToRobot() + " angle");
+		System.out.println(shooterSensors.getShooterSpeed() + " speed");
+		try {
+			Thread.sleep(100);
+		} catch (Exception e) {
+			//TODO: handle exception
+		}
+		//System.out.println(shooterSensors.getShooterAngleToRobot() + " angle");
+		SmartDashboard.putNumber("shooter angle", shooterSensors.getShooterAngleToRobot());
```
Comment (secretspy97) [inline src/main/java/frc/subsystem/Shooter.java]: REMOVE

File: src/main/java/frc/subsystem/Storage.java [hunk 2]
```diff
@@ -44,13 +44,14 @@ void execute() {
...
 			//Set Motor Values
-			motorMaster.set(ControlMode.PercentOutput, output.getOutput());
+			motorMaster.set(ControlMode.PercentOutput, -output.getOutput());
 			if(output.isCommandFinished()) {
 				//TURN OFF MOTORS
 				motorMaster.set(ControlMode.PercentOutput, 0);
 				storageCommand = null;
 			}
 		}
+		motorMaster.set(ControlMode.PercentOutput, -0.5);
```
Comment (secretspy97) [inline src/main/java/frc/subsystem/Storage.java]: REMOVE

File: src/main/java/frc/subsystem/Shooter.java [hunk 3]
```diff
@@ -37,6 +39,19 @@ void execute() {
  			FlywheelMotorAlpha.set(ControlMode.PercentOutput, shooterOutput.getOutputValue());
  			FlywheelMotorBeta.set(ControlMode.PercentOutput, shooterOutput.getOutputValue());
 		} 
+		FlywheelMotorAlpha.set(ControlMode.PercentOutput, -0.64);
```
Comment (secretspy97) [inline src/main/java/frc/subsystem/Shooter.java]: REMOVE

Comment (secretspy97) [reply src/main/java/frc/subsystem/Shooter.java]: and below

File: src/main/java/frc/subsystem/Shooter.java [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
```
Comment (secretspy97) [inline src/main/java/frc/subsystem/Shooter.java]: Remove unused import

File: src/main/java/frc/subsystem/Acquisitions.java [hunk 5]
```diff
@@ -4,19 +4,20 @@
 import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 
 import frc.acq.AcqCommand;
+import frc.acq.IntakeRollers;
 import frc.robot.IO;
 
 public class Acquisitions extends Subsystem{
 
-	private AcqCommand acqCommand;
+	private AcqCommand acqCommand = new IntakeRollers();
```
Comment (secretspy97) [inline src/main/java/frc/subsystem/Acquisitions.java]: shouldn't be set

**Review by secretspy97 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/subsystem/Shooter.java [hunk 6]
```diff
@@ -37,7 +39,9 @@ void execute() {
  			FlywheelMotorAlpha.set(ControlMode.PercentOutput, shooterOutput.getOutputValue());
  			FlywheelMotorBeta.set(ControlMode.PercentOutput, shooterOutput.getOutputValue());
 		} 
-	}
+		FlywheelMotorAlpha.set(ControlMode.PercentOutput, -0.64);
+		FlywheelMotorBeta.set(ControlMode.PercentOutput, 0.64);
```
Comment (secretspy97) [inline src/main/java/frc/subsystem/Shooter.java]: Remove

**Review by secretspy97 — APPROVED** [review]

**Review by NickRicotta — APPROVED** [review]

**Pull request merged**